extensions/elytron-security-ldap/deployment/src/test/java/io/quarkus/elytron/security/ldap/CacheTest.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.elytron.security.ldap;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.elytron.security.ldap.rest.SingleRoleSecuredServlet;
+import io.quarkus.test.QuarkusUnitTest;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.ldap.LdapServerTestResource;
+import io.restassured.RestAssured;
+
+@QuarkusTestResource(LdapServerTestResource.class)
+public class CacheTest {
```

Comment:
Problem with this test is that it would pass even on the current main where there is no caching feature. I think proper test would require stopping the `LdapServerTestResource`. That however, would require to implement `io.quarkus.test.common.QuarkusTestResourceLifecycleManager#inject(io.quarkus.test.common.QuarkusTestResourceLifecycleManager.TestInjector)`, injecting it into this test, stop the LDAP server and make request again.

How does it sound? As Sergey already approved it is not required ...

Reply:
@michalvavrik It is required because you noticed it :-). I approved as I thought the actual code change done by @jonasjensen-brolstark was good and simple, but I was delaying  enabling the CI run since your approval is also required for this PR. Thanks for the review so far

Reply:
Good catch. I will try to give it a try and implement that test.

Reply:
Thank you!